bigtable/google/cloud/bigtable/row.py

```diff
@@ -236,6 +246,12 @@ def _delete_cells(self, column_family_id, columns, time_range=None,
             mutations_list.extend(to_append)
 
 
+def call_mutate_row(table, request_pb):
```

Comment:
Couldn't you just use `self._table._instance._client._data_stub.MutateRow` directly below?

Reply:
Please see the reply above about the use of a callable wrapped partial.